src/client/search/searchReducer.spec.js

```diff
@@ -138,14 +149,15 @@ describe('#SearchTermsReducer', () => {
 
   it('should add a team entity search term', () => {
     const stateBefore = [];
-    const action = actions.addSearchTerm('#Manchester United FC', actions.TEAM_ENTITY);
+    const action = actions.addSearchTerm('#Manchester United FC', actions.TEAM_ENTITY, 1);
```

Comment:
Instead of 1 as a magic number, could it at least be a constant called like EXAMPLE_DEFAULT_DETAILS?